src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -463,15 +478,32 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ReleaseModifyResponse'
-                $ref: '#/components/schemas/ReleasePinNotSetResponse'
+                anyOf:
+                  - $ref: '#/components/schemas/ReleaseModifyResponse'
+                  - $ref: '#/components/schemas/ReleasePinNotSetResponse'
         '400':
           description: Client error
         '403':
           description: Permission denied
 
   /pins/{product}/{channel}/{version}:
     get:
```

Comment:
When I loaded the API spec into the Swagger editor it found a few semantic errors on this end-point because "Declared path parameter "foo" needs to be defined as a path parameter at either the path or operation level"